src/main/java/org/fundacion/jala/converter/models/User.java

```diff
@@ -0,0 +1,147 @@
+
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package org.fundacion.jala.converter.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.CascadeType;
+import javax.persistence.OneToMany;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "USERS")
+public class User {
+
+    @Id
+    @Column(name = "USER_ID")
+    private int id;
+
+    @Column(name = "NAME")
+    private String name;
+
+    @Column(name = "PASWORD")
+    private String pasword;
+
+    @Column(name = "TOKEN")
+    private String token;
+
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
+    private List<Project> projects = new ArrayList<>();
+
+    public User() {
+
+    }
+
+    public User(final int newId, final String newName, final String newPasword, final String newToken) {
+        this.id = newId;
+        this.name = newName;
+        this.pasword = newPasword;
+        this.token = newToken;
+    }
+
+    /**
+     * obtains Ids of the User
+     * @return id of the user
+     */
+    public int getId() {
+        return id;
+    }
+
+    /**
+     * sets new Id of the User
+     * @param newId
+     */
+    public void setId(final int newId) {
+        this.id = newId;
+    }
+
+    /**
+     * Obtains Name of the User
```

Comment:
Perhaps the Javadoc should be in singular.

Reply:
ok